packages/protoplugin-example/app/src/gen/buf/connect/demo/eliza/v1/eliza_twirp.d.ts

```diff
@@ -0,0 +1,63 @@
+// Copyright 2021-2022 Buf Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// @generated by protoc-gen-twirp-es v0.1.0 with parameter "target=ts+js+dts"
```

Comment:
It doesn't make much sense to generate both `.js` and `.ts` files. I'm not even sure which file is going to win for the compiler. Considering we are using `target=ts` for protobuf-example, it seems reasonable to use here as well.